pkg/standalone-utils/test/LidoRelayer.test.ts

```diff
@@ -1,462 +1,950 @@
-            await relayer.connect(sender).batchSwap(SwapKind.GivenIn, swaps, assets, funds, limits, deadline);
+          const receipt = await (
+            await relayer.connect(sender).multicall([encodeStakeETH(tokenRecipient, amount)], { value: amount })
+          ).wait();
 
-            expect(await WETH.balanceOf(relayer)).to.be.eq(0);
-            expect(await wstETH.balanceOf(relayer)).to.be.eq(0);
-          });
+          const relayerIsRecipient = TypesConverter.toAddress(tokenRecipient) === relayer.address;
+          expectTransferEvent(
+            receipt,
+            {
+              from: relayerIsRecipient ? ZERO_ADDRESS : relayer.address,
+              to: relayerIsRecipient ? relayer.address : TypesConverter.toAddress(tokenRecipient),
+              value: amount,
+            },
+            stETH
+          );
         });
 
-        describe('swap using stETH as an output', () => {
-          sharedBeforeEach('build swap request', async () => {
-            swaps = [
-              {
-                poolId: basePoolId,
-                assetInIndex: basePoolTokens.findIndexBySymbol('WETH'),
-                assetOutIndex: basePoolTokens.findIndexBySymbol('wstETH'),
-                amount: fp(1),
-                userData: '0x',
-              },
-            ];
-
-            assets = basePoolTokens.addresses;
-            limits = basePoolTokens.map((token) => (token.symbol === 'wstETH' ? 0 : fp(1)));
-          });
+        it('stores stake output as chained reference', async () => {
+          const amount = fp(1);
 
-          it('performs the given swap', async () => {
-            const receipt = await relayer
+          const receipt = await (
+            await relayer
               .connect(sender)
-              .batchSwap(SwapKind.GivenIn, swaps, assets, funds, limits, deadline);
-
-            expectEvent.inIndirectReceipt(await receipt.wait(), vault.instance.interface, 'Swap', {
-              poolId: swaps[0].poolId,
-              tokenIn: assets[swaps[0].assetInIndex],
-              tokenOut: assets[swaps[0].assetOutIndex],
-              amountIn: swaps[0].amount,
-              // amountOut
-            });
-          });
+              .multicall([encodeStakeETH(tokenRecipient, amount, toChainedReference(0))], { value: amount })
+          ).wait();
 
-          it('does not leave dust on the relayer', async () => {
-            await relayer.connect(sender).batchSwap(SwapKind.GivenIn, swaps, assets, funds, limits, deadline);
+          const relayerIsRecipient = TypesConverter.toAddress(tokenRecipient) === relayer.address;
+          expectTransferEvent(
+            receipt,
+            {
+              from: relayerIsRecipient ? ZERO_ADDRESS : relayer.address,
+              to: relayerIsRecipient ? relayer.address : TypesConverter.toAddress(tokenRecipient),
+              value: amount,
+            },
+            stETH
+          );
```

Comment:
```suggestion
```
This is the same assertion as the one in the previous test, seems we can drop it?